tests/grpc/grpc_test.go

```diff
@@ -0,0 +1,499 @@
+// Copyright 2021 ChainSafe Systems
+// SPDX-License-Identifier: MIT
+
+package grpc
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/ChainSafe/chainlink-cosmos/app"
+	testnet "github.com/ChainSafe/chainlink-cosmos/testutil/network"
+	"github.com/ChainSafe/chainlink-cosmos/x/chainlink/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/config"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	"github.com/cosmos/cosmos-sdk/testutil/testdata"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	signing2 "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	rpcclient "github.com/tendermint/tendermint/rpc/client"
+	"google.golang.org/grpc"
+)
+
+type testAccount struct {
+	Name   string
+	Priv   cryptotypes.PrivKey
+	Pub    cryptotypes.PubKey
+	Addr   sdk.AccAddress
+	Cosmos string
+}
+
+var (
+	//alice = generateKeyPair("alice")
+	//bob   = generateKeyPair("bob")
+	//cerlo = generateKeyPair("cerlo")
+	alice *testAccount
+	bob   *testAccount
+	cerlo *testAccount
+)
+
+//func generateKeyPair(name string) *testAccount {
```

Comment:
is it safe to remove those commented funcs?

Reply:
I prefer to let's the comment part here for the future if we find a way to have test running with `testnet`
If we want to keep the current solution, yes we can remove the commented code

Reply:
no it's totally ok to keep it so that we can save time for the future. Maybe you can leave a TODO there mention that this is for the future in case we forgot.